relay-metrics/src/aggregation.rs

```diff
@@ -1,11 +1,12 @@
-use std::cmp::max;
-use std::collections::{btree_map, hash_map::Entry, BTreeMap, BTreeSet, HashMap};
-
-use std::fmt;
-use std::hash::Hasher;
-use std::iter::{FromIterator, FusedIterator};
-use std::mem;
-use std::time::{Duration, Instant};
+use std::{
+    cmp::max,
+    collections::{btree_map, hash_map::Entry, BTreeMap, BTreeSet, HashMap},
+    fmt,
+    hash::Hasher,
+    iter::{FromIterator, FusedIterator},
+    mem,
+    time::{Duration, Instant},
+};
```

Comment:
:( can we not change this import style?

Reply:
I might went overboard with this imports here 😅  I can revert this back.